powerline/segments/common/time.py

```diff
@@ -3,21 +3,34 @@
 
 from datetime import datetime
 
-
-def date(pl, format='%Y-%m-%d', istime=False):
+def date(pl, format='%Y-%m-%d', istime=False, tz=None):
 	'''Return the current date.
 
 	:param str format:
 		strftime-style date format string
 	:param bool istime:
 		If true then segment uses ``time`` highlight group.
+	:param string tz:
+		Timezone parseable by pytz (requires ``pytz`` module).
 
 	Divider highlight group used: ``time:divider``.
 
 	Highlight groups used: ``time`` or ``date``.
 	'''
+
+	if (tz):
+		try:
+			from pytz import timezone, utc
```

Comment:
I would highly suggest importing at the top level and setting `None` in case import fails (it fails with `ImportError` and only this error should be caught) because otherwise this function is not testable.